backend/app/api/endpoints/openapi_responses.py

```
@@ -294,95 +294,481 @@ async def create_response(
+        task_id,
+        api_key_name,
+    )
+
+    response_id = f"resp_{setup.task_id}"
+    created_at = int(datetime.now().timestamp())
+    assistant_subtask_id = setup.assistant_subtask.id
+    task_kind_id = setup.task_id
+    enable_chat_bot = tool_settings.get("enable_chat_bot", False)
+
+    # Extract data needed for streaming before closing db
+    user_id = user.id
```

Ruff (0.14.14)

[error] Local variable `user_id` is assigned to but never used

Remove assignment to unused variable `user_id`

(F841)